### ISSUE TITLE
Monokai Extended Light

### ISSUE THREAD

**Issue by mcnesium** [jonschlinkert/sublime-markdown-extended#80]
Using Monokai Extended Light theme GFM fenced blocks look like this:

![tisuuth1](https://cloud.githubusercontent.com/assets/476060/7042624/811d0cf8-dde3-11e4-8892-1e6247ddfea8.png)

The background color should be adjusted to light background color.

**Comment by jonschlinkert** [comment]
yeah the light theme needs to be updated. Honestly I think I just forgot to update it when I made changes a long time ago, but it seems like most users want the dark them. I personally don't use the light theme, but I'll work on getting it updated when I get back from vacation next week.  

pr would be welcome too!

**Comment by mcnesium** [comment]
I like the light theme better in my daylight-flooded office ;)

When I figure out how to modify ST themes, I'll be happy to file a PR.

**Comment by wgregp** [comment]
To fix, at `Monokai Extended Light.tmtheme` line 1508 set the background string to `#fafafa`. You can also do the same thing in the `JSON-tmtheme` at line 939.
